book/src/pong-tutorial/pong-tutorial-01.md

```diff
@@ -77,25 +95,21 @@ Inside `main()` we first start the amethyst logger with a default `LoggerConfig`
 so we can see errors, warnings and debug messages while the program is running.
 
 ```rust,edition2018,no_run,noplaypen
-# extern crate amethyst;
-# use amethyst::prelude::*;
-# fn main() -> amethyst::Result<()> {
-amethyst::start_logger(Default::default());
```

Comment:
I'm not sure you can just delete this, as tests for the book won't pass any more without these imports.

Reply:
What command do I run to check these? Is it part of that `mdbook` command I cargo installed?

Reply:
Ah, you need to use

`mdbook test book -L target/debug/deps`

Make sure you clean & re-build before, otherwise it fails to resolve dependencies correctly.